src/App.js

```diff
@@ -4,94 +4,182 @@ import styled from 'styled-components';
 import { confluenceLinks } from './constants';
```

Comment:
I can't comment on line 1 because there's no changes there but as we're no longer using the `useState` function we should delete this from the imports